cetmix_tower_yaml/static/description/index.html

```diff
@@ -435,32 +460,72 @@ <h2><a class="toc-backref" href="#toc-entry-7">17.0.1.0.1 (2025-05-05)</a></h2>
 </ul>
 </div>
 <div class="section" id="section-5">
+<<<<<<< HEAD
```

Comment:
This is left from merge conflict resolve, you must resolve the conflicts properly